Backend/app/dashboard/Lib/src/Domain/Types/Transaction.hs

```diff
@@ -14,6 +14,14 @@
 
 module Domain.Types.Transaction where
 
+import qualified "rider-app" API.Dashboard.RideBooking.Booking as BAP
+import qualified "rider-app" API.Dashboard.RideBooking.Confirm as BAP
+import qualified "rider-app" API.Dashboard.RideBooking.Maps as BAP
+import qualified "rider-app" API.Dashboard.RideBooking.Profile as BAP
+import qualified "rider-app" API.Dashboard.RideBooking.Quote as BAP
+import qualified "rider-app" API.Dashboard.RideBooking.Registration as BAP
+import qualified "rider-app" API.Dashboard.RideBooking.Search as BAP
+import qualified "rider-app" API.Dashboard.RideBooking.Select as BAP
```

Comment:
These are newly added files, so updated cabal of rider app should also be pushed